pandas/tests/frame/test_indexing.py

```diff
@@ -2590,3 +2590,19 @@ def test_head_tail(self):
         empty_df = DataFrame()
         assert_frame_equal(empty_df.tail(), empty_df)
         assert_frame_equal(empty_df.head(), empty_df)
+
+    def test_type_error_multiindex(self):
+        df = pd.DataFrame(columns=['i', 'c', 'x', 'y'],
+                          data=[[0, 0, 1, 2], [1, 0, 3, 4],
+                                [0, 1, 1, 2], [1, 1, 3, 4]])
+        dg = df.pivot_table(index='i', columns='c',
+                            values=['x', 'y'])
+
+        try:
+            print(dg[:, 0])
```

Comment:
you can do ``str(dg[:, 0])`` to trigger an error and use an ``.assertRaises`` to test for it.